cEP-0020.md

```diff
@@ -0,0 +1,532 @@
+is building. And a way to use those feedback to improve coala and
+its processes.
+
+## Why Gamification?
+
+As coala tries to be a welcoming organization for newcomers,
+it gives them clear pathways and as much direct assistance as
+possible. However, it is already known that newcomers face many
+barriers while attempting to contribute to open source for the
+first time, and additional barriers while attempting to
+contribute to projects run by the coala organization.
+
+Some barriers they face include orientation issues that can
+potentially demotivate newcomers from placing their first
+contribution. On the other hand, gamification is widely used
+to engage and motivate people to accomplish tasks and improve
+their performance.
+
+Therefore, the goal of this project is to use gamification to
+orient and motivate newcomers to overcome onboarding barriers
+to contribute and engage with coala and its community.
+
+## Gamification Introduction
+
+There will be four game design elements to help newcomers:
+Activities, Points, Badges and Levels. For each of these
+elements, there will be a set of rules to describe the
+operation of the gamified environment.
+
+### Activities
+
+Let's start by defining some activities which could be tracked:
+
+- Create a bug report
+- Create newcomer difficulty issue
+- Create a bear proposal issue
+- Create a documentation issue
+- Create low difficulty issue
+- Create medium difficulty issue
+- Create pull request that fixes a bug
+- Create pull request for bear proposal
+- Create pull request for a documentation issue
+- Create pull request for newcomer issue
+- Create pull request for low difficulty issue
+- Create pull request for medium difficulty issue
+- Gitmate errors on pull request
+- CI status of the pull request
+- Review comments on pull request
+- Number of questions asked in `coala/coala` room
+- Number of answers given in `coala/coala` room
+- Number of questions asked in `coala-bears` room
+- Number of  answers given in `coala-bears` room
+- Number of questions asked in `coala/corobo` room
+- Number of answers given in `coala/corobo` room
+- Number of questions asked in `coala/community` room
+- Number of answers given in `coala/community` room
+- Number of messages in coala other rooms
+- Make use of `corobo` on Gitter
+- Introduce `.coafile` in user own repository
+- Introduce `.coafile` in an organisation with 50 or more stars
+  for its repositories
+- Introduce CI in the users own repository to run coala using
+  Travis CI
+- Setup the Circle CI to the fork of coala repository
+- Setup Travis CI to the fork of coala repository
+- Setup Appveyor CI to the fork of coala repository
+
+### Points
+
+Points are a simple tool to reward positive contributions
+to coala, and they provide an excellent base upon which to
+build a gamified design. Now let's assign points to some
+of the tracked activities mentioned above:
+
+- Create Issue
+  - 20 points for type/bug
+  - 20 points for bear proposal
```

Comment:
bear proposals should be much lower.  those issues are usually just a link to a linter.  very easy to create, and there are hundreds of external linters that coala dont support, and many of them are not better than the bears we already have.

Also , we can split them into 

1. external linter bear proposal, which are wrapping an existing tool = 5 points
2. native bear proposal = 7 points
3. generic bear proposal = 10 points